system_metrics_collector/src/system_metrics_collector/constants.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SYSTEM_METRICS_COLLECTOR__CONSTANTS_HPP_
+#define SYSTEM_METRICS_COLLECTOR__CONSTANTS_HPP_
+
+#include <chrono>
+
+namespace CollectorNodeConstants
```

Comment:
```suggestion
namespace system_metrics_collector
```

To be consistent with the rest of the project: https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
How about `system_metrics_collector::collector_node_constants`? It's a little verbose but that way we can distinguish them more easily.

Reply:
> How about `system_metrics_collector::collector_node_constants`? It's a little verbose but that way we can distinguish them more easily.

SGTM!